jobs/JGFS_ATMOS_CYCLONE_TRACKER

```
@@ -128,8 +128,7 @@ export loopnum=1
 
 #-----------input data checking -----------------
 ${USHens_tracker}/data_check_gfs.sh
-# exit code 6 = missing data of opportunity
-if [ $? -eq 6 ]; then exit; fi
+export err=$?; err_chk
```

Should code 6 still be ignored? If the comment is correct, I don't know that we want to fail the job if data of opportunity is missing but everything else is correct.

---

Can I get an answer to this so we can get this PR in

---

Code 6 here is not consider as failure. It can be ignored to make exception handling effective.

---

Okay, this needs to be updated then to retain that from the previous code.

---

I think this works.